src/components/radioGroup/radio/radio.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { IconType } from '../../icon';
+import { RadioGroup } from '../radioGroup';
+import { Radio, type IRadioProps } from './radio';
+
+describe('Radio', () => {
+    const createTestComponent = (props?: Partial<IRadioProps>) => {
+        const completeProps = { label: 'test label', value: 'test value', ...props };
+
+        return (
+            <RadioGroup name="Test Group">
+                <Radio {...completeProps} />;
+            </RadioGroup>
+        );
+    };
+
+    it('should render a label and an unchecked radio button', () => {
```

Comment:
Just to follow current naming convention on tests:
```suggestion
    it('renders a label and an unchecked radio button', () => {
```
Same for the tests below